### ISSUE TITLE
Data Inputs

### ISSUE THREAD

**Issue by Jaclye** [yzshi5/GM-GANO#8]
Hello, author.
I see that the order in which the images are generated is E N Z. I should enter them in that order when making the numpy array right?

**Comment by yzshi5** [comment]
Yes, the training dataset have a shape of [N_records, 3, duration * sampling_frequency], the `3` means `E, N, Z` components. You should keep such order for your training dataset. The model automatically learns such order when you use the trained model for generating waveforms

**Comment by Jaclye** [comment]
Hi author, does filtering or baseline correction come first in data processing?

**Comment by yzshi5** [comment]
Yes, preprocessing methods are needed. A high quality dataset is crucial for training the model. Please refer to closed question #4 and question #5 for more information about preprocessing pipeline
